spec/spec_helper.rb

```diff
@@ -60,7 +60,7 @@
   #
   #   # Limits the available syntax to the non-monkey patched syntax that is
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly `index` and `show` actions for `User` and `Post` controllers should be tested 🙏🏼 

```ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

![image](https://github.com/tamoorsaeed22/blog_app_project/assets/63915024/bce06471-e3e6-49ca-abd1-f2b85f686960)